docs/using/authenticating-to-github.md

```diff
@@ -0,0 +1,35 @@
+# Authenticating to GitHub
+
+## How to login to GitHub
+
+1. In Unity find the **GitHub** window. If you can't find it, you can switch to it by choosing **GitHub** from the **Window** menu.
```

Comment:
This is not as straightforward as it could be. Just saying "Open the **_GitHub_** window by going to the top level **Window** menu and selecting **GitHub**, as shown below" might work better.

Reply:
👍